src/app/layout.tsx

```diff
@@ -27,7 +27,7 @@ export default function RootLayout({
           <div className="bg-background">
             <div className="flex w-full min-h-screen">
               <Sidebar className="w-[300px] hidden lg:block border-r" />
-              <div className="p-5 mx-auto w-[1300px]">{children}</div>
+              {children}
             </div>
```

Comment:
We don't need sidebar here because login page will be affected. If user can't pass authentication, user can't see our menu page. Next time I will indicate in issue _(this is lack of information)_

```suggestion
       <div className="flex w-full min-h-screen justify-center">
              {children}
       </div>
```